Cargo.toml

```diff
@@ -40,14 +40,14 @@ async-trait = "0.1"
 bitmask-enum = "2.2.4"
 chrono = "0.4.31"
 crossbeam-channel = "0.5.12"
-env_logger = "0.10.0"
 lazy_static = "1.4.0"
-log = "0.4.17"
 prost = "0.12"
 prost-types = "0.12"
 protobuf = { version = "3.3" }
 rand = "0.8.5"
 tokio = { version = "1.35.1", default-features = false }
+tracing = "0.1.40"
```

Comment:
I realized as well that this is the superior crate to use for multi-threaded and very-async libraries

Want to do the same for up-transport-vsomeip-rust and up-streamer-rust in a later PR :slightly_smiling_face: